apps/web/src/components/mapping/google-maps/marker-info-box.tsx

```
@@ -0,0 +1,239 @@
+          <MarkerPin
+            color={activeLocation.color}
+            icon={activeLocation.icon as IconType}
+            size={28}
+          />
+        </div>
+
+        {/* Info Card (always visible) */}
+        <div className="absolute bottom-8 left-1/2 -translate-x-1/2 w-[200px]">
+          <div className="relative overflow-hidden rounded-md bg-white shadow-xl border border-gray-200">
+            {/* Image */}
+            {activeLocation?.place.photos && activeLocation?.place.photos[0] ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)